webapp/src/dogma/common/components/editor/JsonPath.tsx

```diff
@@ -0,0 +1,11 @@
+import { DebouncedInput } from 'dogma/common/components/table/DebouncedInput';
+import { Box, Text } from '@chakra-ui/react';
+
+export const JsonPath = ({ handleQuery }: { handleQuery: (value: string | number) => void }) => {
+  return (
+    <Box mb="4">
+      <Text mb="2">Query by JSON Path</Text>
```

Comment:
`JSONPath` seems like an official term.
- https://goessner.net/articles/JsonPath/
- https://www.baeldung.com/guide-to-jayway-jsonpath
```suggestion
      <Text mb="2">Query by JSONPath</Text>
```